content/doc/pipeline/declarative/Advanced.adoc

```diff
@@ -0,0 +1,105 @@
+The pipeline you have seen so far has been mostly declarative. You can
```

Comment:
Did you mean for this to be just a few grammar fixes?  If so, you added quite a bit more.